src/components/Menu/components/MenuAction/MenuAction.scss

```diff
@@ -87,18 +83,3 @@ $action-horizontal-padding: (1.5 * spacing(tight));
     @include recolor-icon(color('ink', 'lightest'));
   }
 }
-
-.Action-outline {
-  @include button-base;
-  @include button-outline(color('ink', 'lighter'));
-
-  &::after {
-    top: auto;
-    left: auto;
-    right: auto;
-  }
-
-  &.disabled {
-    @include button-outline-disabled(color('ink', 'lighter'));
-  }
-}
```

Comment:
Also curious why this gets removed

Reply:
`outline` was never a variant this component accepted